bakery/src/tasks/upload-single.js

```diff
@@ -0,0 +1,62 @@
+const { constructImageSource } = require('../task-util/task-util')
+const fs = require('fs')
+const path = require('path')
+
+const task = (taskArgs) => {
+  const { awsAccessKeyId, awsSecretAccessKey, distBucket, codeVersion, distBucketPath } = taskArgs
+  const imageDefault = {
+    name: 'openstax/cops-bakery-scripts',
+    tag: 'trunk'
+  }
+  const imageOverrides = taskArgs != null && taskArgs.image != null ? taskArgs.image : {}
+  const imageSource = constructImageSource({ ...imageDefault, ...imageOverrides })
+  if (!distBucketPath.endsWith('/') || distBucketPath.length === 0) {
+    throw Error('distBucketPath must represent some directory-like path in s3')
+  }
+  const distBucketPrefix = `${distBucketPath}${codeVersion}`
+  const bookInput = 'book'
+  const jsonifiedInput = 'jsonified-single'
+  const checksumInput = 'checksum-single'
```

Comment:
When you try to merge, this output will no longer exist. `checksum-single` has been removed on the main branch. There is now a `resources` output with all the checksummed resources in it. You should be able to treat it identically.

Reply:
I'd like to merge these changes at this time, and have another issue to account for the changes (https://github.com/openstax/cnx/issues/1317) needed to make to the pipeline to account for the checksum changes - this will allow me to work with a more updated tag.